service-front/assets/sass/patterns/_help-system.scss

```diff
@@ -185,6 +185,12 @@
 .help-footer {
     padding: $gutter 0 0;
 
+    p {
+        overflow: hidden;
```

Comment:
nice bit of CSS - how i miss messing with this stuff 😁

Reply:
I could probably just add break word and not the other styles, but these don't hurt as they only affect this one paragraph and help it display better where the word breaks aren't supported (e.g. very old browsers).

Reply:
makes sense